src/System.Net.Primitives/tests/FunctionalTests/CookieCollectionTest.cs

```diff
@@ -108,6 +108,31 @@ public static void Add_CookieCollection_Invalid()
             Assert.Throws<ArgumentNullException>(() => cc.Add((CookieCollection)null));
         }
 
+        [Fact]
```

Comment:
It looks like the tests have a configuration that builds against netstandard.  As such, these tests won't compile in that configuration because these new members don't exist in that configuration.  To add the tests, you'll need to move them into a separate file, e.g. CookieCollectionTest.netcoreapp.cs, and then change the .csproj to conditionally include that file only when building for netcoreapp.  If you search the repo, you'll find lots of other examples of that you can copy from.